index.html

```diff
@@ -28,6 +28,111 @@ <h1>I'm Ahmed <br> Glad to see <br> you!</h1>
       <li class="icons"><a href="#"><img src="./svgs/Vector(2).svg" alt="svg"></a></li>
       <li class="icons"><a href="#"><img src="./svgs/Vector(3).svg" alt="svg"></a></li>
     </ul>
+  </section><br>
+  <section class="works">
+    <div class="card1">
+      <img src="works/Snapshoot-Portfolio1.svg" alt="snaphot" class="p-img">
+      <h2>Tonic</h2>
+      <section class="u-c">
+        <h5>CANOPY</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>Back End Dev</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>2015</h5>  
+      </section>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="u-html">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <a href="#"><h4>See Project</h4></a>
+    </div>
+    <div class="card2">
+      <img src="works/Snapshoot-Portfolio2.svg" alt="snaphot" class="p-img">
+      <h2>Multi-Post <br>Stories</h2>
+      <section class="u-c">
+        <h5>CANOPY</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>Back End Dev</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>2015</h5>  
+      </section>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="u-html">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <a href="#"><h4>See Project</h4></a>
+    </div>
+    <div class="card3">
+      <img src="works/Snapshoot-Portfolio3.svg" alt="snaphot" class="p-img">
+      <h2>Tonic</h2>
+      <section class="u-c">
+        <h5>CANOPY</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>Back End Dev</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>2015</h5>  
+      </section>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="u-html">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <a href="#"><h4>See Project</h4></a>
+    </div>
+    <div class="card4">
+      <img src="works/Snapshoot-Portfolio4.svg" alt="snaphot" class="p-img">
+      <h2>Multi-Post <br>Stories</h2>
+      <section class="u-c">
+        <h5>CANOPY</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>Back End Dev</h5>
+        <img src="works/Counter.svg" alt="counter">
+        <h5>2015</h5>  
+      </section>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="u-html">
+        <li>html</li>
+        <li>css</li>
+        <li>javascript</li>
+      </ul>
+      <a href="#"><h4>See Project</h4></a>
+    </div>      
+  </section>
+  <section class="about-section">
```

Comment:
- [ ] please try to stick with the Figma design as much as possible currently there are missing arrows images and wrong margins, paddings, font sizes and colors . kindly try to fix them all to make the section looks great

![fix it](https://user-images.githubusercontent.com/89282221/185711745-85d831aa-9a81-46d6-b9a9-7a41576c6b7a.png)